spec/models/spree/calculator/flexi_rate_spec.rb

```diff
@@ -1,13 +1,18 @@
 require 'spec_helper'
 
 describe Spree::Calculator::FlexiRate do
-  let(:calculator) { Spree::Calculator::FlexiRate.new }
+  let(:calculator) { Spree::Calculator::FlexiRate.new(preferred_first_item: 2, preferred_additional_item: 1) }
   let(:line_item) { instance_double(Spree::LineItem, amount: 10, quantity: 4) }
 
   describe "computing for a single line item" do
-    it "returns the first item rate" do
-      calculator.stub preferred_first_item: 1.0
-      calculator.compute(line_item).round(2).should == 1.0
+    it "returns the first item rate when above max" do
+      calculator.stub preferred_max_items: 3.0
```

Comment:
I just noticed this. We shouldn't stub the class under test :red_circle: . Can't we simply build a new calculator instance with this atttribute set to 3.0?

Reply:
Sure